minecraft_servers/novacraft/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+    "server_name": "novacraft",
+    "nice_name": "NovaCraft",
+    "direct_ip": "Novacraftmc.de",
+    "supported_languages": [
+        "de"
+    ],
+    "social": {
+        "tiktok": "novacraftmc.de",
+        "discord": "https://dsc.gg/novacraftmc",
```

Comment:
Sadly, we cannot allow those links
Please use https://discord.gg/[INVITE]

Reply:
I did it